src/DssSpell.sol

```diff
@@ -37,7 +81,41 @@ contract DssSpellAction is DssAction {
     // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d8c711404e9e17db24b611aa03d45507c1993148/governance/votes/Executive%20vote%20-%20November%2016%2C%202022.md -q -O - 2>/dev/null)"
 
     string public constant override description =
-        "2022-11-16 MakerDAO Executive Spell | Hash: 0xf3023b0a9b3c423edf28bf4cdbd61d287ab29265d462a5fae0378227c153ea7b";
+        "2022-11-30 MakerDAO Executive Spell | Hash: TODO";
+
+    uint256 constant internal RAY = 10 ** 27;
+    uint256 constant internal MILLION = 10 ** 6;
+
+    bytes32 constant internal ILK = "DIRECT-COMPV2-DAI";
+
+    address constant internal D3M_HUB = 0x12F36cdEA3A28C35aC8C6Cc71D9265c17C74A27F;
+    address constant internal D3M_MOM = 0x1AB3145E281c01a1597c8c62F9f060E8e3E02fAB;
+    address immutable internal D3M_MOM_LEGACY = DssExecLib.getChangelogAddress("DIRECT_MOM");
+    address constant internal D3M_COMPOUND_POOL = 0x621fE4Fde2617ea8FFadE08D0FF5A862aD287EC2;
+    address constant internal D3M_COMPOUND_PLAN = 0xD0eA20f9f9e64A3582d569c8745DaCD746274AEe;
+    address constant internal D3M_ORACLE = 0x0e2bf18273c953B54FE0a9dEC5429E67851D9468;
```

Comment:
Not requesting a change here but this should have been named D3M_COMP_ORACLE since there are both Comp-specific config and D3M-wide config actions in this spell (I was confused about what D3M_ORACLE was for a while). Just my feedback

Reply:
Yeah it's a fair point. Something to keep in mind for next time.

Reply:
In general I'm a fan to use the same text for the constants name than the chainlog key (even if they are large). It is even easier for reviewing. But anyway, we are fine in the meanwhile the chainlog key is correct.